expression/builtin_time.go

```diff
@@ -1770,6 +1770,60 @@ func getTimeZone(ctx context.Context) *time.Location {
 	return ret
 }
 
+//isDuration returns a boolean indicating whether the str is duration
+func isDuration(str string) bool {
```

Comment:
@XuHuaiyu 'isDuration' will be removed to 'util/types/time.go' and a regular expression will be  completed in the new PR